projexui/widgets/xviewwidget/xviewpanel.py

```diff
@@ -1481,7 +1523,7 @@ def switchCurrentView(self, viewType):
         # create the new view
         index = self.indexOf(view)
         if not view.close():
-            return False
```

Comment:
Eric, please make sure this is ok and that other places are not dependent on the False.